src/components/events/createEvent/createEvent.tsx

```diff
@@ -70,7 +70,7 @@ const CreateEvent = () => {
     setAttr('type', data.type);
```

Comment:
I cant commect on line 64 so: 

I prefer  ... to Object.assign
``` 
 const setAttr = (attr: keyof IEvent, value: any) => {
    setNewEvent(...newEvent, { [attr]: value }));
  };
```

Reply:
setNewEvent({...newEvent, [attr]: value }));

Reply:
Done